chart/px-central/values.yaml

```diff
@@ -186,7 +186,7 @@ images:
     registry: icr.io/ext
     repo: portworx
     imageName: keycloak-login-theme
-    tag: 2.7.0
+    tag: 2.7.1
```

Comment:
keycloak-login-theme image with tag 2.7.1 doesn't exist. you can try downloading from docker.io/portworx/keycloak-login-theme:2.7.1 you might want to change it back to 2.7.0